### PR TITLE
chore: update all dependencies and fix few issues

### PR DESCRIPTION
**Pull request by mrmartineau**
This PR updates the project dependencies and fixes any bug that arose while doing it.

This PR is basically the same as https://github.com/LBHackney-IT/mat-process-homecheck/pull/36 but for this project

### CODE REVIEW

File: storage/migrations/schema/external/from0.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import { Upgrade } from "remultiform/database";
 import ExternalDatabaseSchema from "../../../ExternalDatabaseSchema";
 
-export default (upgrade: Upgrade<ExternalDatabaseSchema["schema"]>): void => {
+const upgradeSchema = (
+  upgrade: Upgrade<ExternalDatabaseSchema["schema"]>
+): void => {
```
Comment (LBHABEBIC) [inline storage/migrations/schema/external/from0.ts:6]: just out of curiosity- why this change?

Comment (LBHABEBIC) [reply storage/migrations/schema/external/from0.ts:6]: is it just a refactor?

**Review by LBHABEBIC — APPROVED** [review]

File: storage/migrations/schema/external/from0.ts (hunk 1 above)
Comment (mrmartineau) [reply storage/migrations/schema/external/from0.ts:6]: no, this was highlighted when I updated next.js, here's the error 
image
